test/units/jwt_sessions/test_authorization.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+require "minitest/autorun"
+require "jwt_sessions"
+
+class TestAuthorization < Minitest::Test
+  include JWTSessions::Authorization
+
+  def test_payload_when_token_is_nil
```

Comment:
the test cases are flaky. Could you please add `setup` method specifying the signing key, so it won't depend on order.

Also it'd be fantastic if you could add a sample action using the payload to the dummy app https://github.com/tuwukee/jwt_sessions/blob/main/test/support/dummy_sinatra_api/app.rb
and a test case covering it for auth/unauth requests
https://github.com/tuwukee/jwt_sessions/blob/main/test/support/dummy_sinatra_api/spec/app_spec.rb